packages/rocketchat-livechat/config.js

```diff
@@ -129,4 +129,15 @@ Meteor.startup(function() {
 		public: true,
 		i18nLabel: 'Apiai_Language'
 	});
+
+	RocketChat.settings.add('Livechat_Routing_Method', 'Least_Amount', { 
+		type: 'select', 
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2726498732/issues/source?bid=3199634&fileBranchId=3348698#l134)